### PR TITLE
docs: add nakama and evm docker image version on world.toml explanation

### PR DESCRIPTION
**Pull request by zulkhair**
Closes: WORLD-1274

## Overview

Add explanation in world.toml to use specific nakama and evm docker image using `NAKAMA_IMAGE`, `NAKAMA_IMAGE_PLATFORM`, `EVM_MAGE`, and `EVM_IMAGE_PLATFORM`

## Brief Changelog

add section on `/cardinal/game/configuration/nakama` and `/cardinal/game/configuration/evm`

## Summary by CodeRabbit

- **New Features**
	- Introduced new configuration parameters for EVM: `EVM_IMAGE` and `EVM_IMAGE_PLATFORM`, enhancing user customization options.
	- Added new configuration options for Nakama: `NAKAMA_IMAGE` and `NAKAMA_IMAGE_PLATFORM`, allowing users to specify image versions and platforms. 

- **Documentation**
	- Updated documentation to include descriptions and examples for the new configuration parameters for both EVM and Nakama.

### CODE REVIEW

**Comment by zulkhair** [comment]
* **#813** <a href="https://app.graphite.dev/github/pr/Argus-Labs/world-engine/813?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/Argus-Labs/world-engine/813?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: docs/cardinal/game/configuration/evm.mdx [hunk 1]
```diff
@@ -87,4 +89,23 @@
 **Example**
 ```
 FAUCET_ENABLED = false
+```
+
+### EVM_IMAGE
+
+Set evm to a specific version. this value is used by world-cli
+
```
[the remark this answers is not in the document]
Comment (ezavada) [reply docs/cardinal/game/configuration/evm.mdx:97]: And maybe also note that for Cardinal v1.6.x and earlier, the image MUST be v1.4.1 or earlier

File: docs/cardinal/game/configuration/nakama.mdx [hunk 2]
```diff
@@ -65,4 +67,23 @@ Prometheus scraping interval in seconds.
...
 NAKAMA_METRICS_INTERVAL = 30
+```
+
+### NAKAMA_IMAGE
+
+Set nakama to a specific version. this value is used by world-cli
+
+**Example**
+```
+NAKAMA_IMAGE = ghcr.io/argus-labs/world-engine-nakama:1.29
```
[the remark this answers is not in the document]
Comment (ezavada) [reply docs/cardinal/game/configuration/nakama.mdx:78]: And maybe also note that for Cardinal v1.6.x and earlier, the image MUST be v1.2.9 or earlier.

[the remark this answers is not in the document]
Comment (zulkhair) [reply docs/cardinal/game/configuration/nakama.mdx:78]: updated

File: docs/cardinal/game/configuration/evm.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (zulkhair) [reply docs/cardinal/game/configuration/evm.mdx:97]: updated

**Review by rmrt1n — APPROVED** [review]

**Pull request merged**